mlxtend/frequent_patterns/fpgrowth.py

```diff
@@ -42,6 +44,9 @@ def fpgrowth(df, min_support=0.5, use_colnames=False, max_len=None, verbose=0):
       The support is computed as the fraction
       transactions_where_item(s)_occur / total_transactions.
 
+    null_values : bool (default: True)
```

Comment:
```suggestion
    null_values : bool (default: False)
```

I think this should say `False` based on the function signature above. (This is a great feature, but `False` is probably also the right choice for backward compatibility.)